packages/office-ui-fabric-react/src/components/Button/BaseButton.tsx

```diff
@@ -181,6 +181,7 @@ export class BaseButton extends BaseComponent<IBaseButtonProps, IBaseButtonState
       className: this._classNames.root,
       ref: this._buttonElement,
       disabled: isPrimaryButtonDisabled && !allowDisabledFocus,
+      onKeyDown: this._onKeyDown,
```

Comment:
You're overriding a user's onKeyDown (if they provide one). You could either not hook up this keyDown handler if !disabled or call this.props.onKeyDown inside of _onKeyDown if it exists and you are not cancelling the event. The first option will make an existing keydown handler behave the same as it does if allowDisabledFocus is false (since the keydown will not fire since the button is not focusable)

Reply:
Got rid of the override.